SublimeLinter.sublime-settings

```diff
@@ -40,6 +40,10 @@
     // you save the buffer.
     "highlights.time_to_idle": 1.5,
 
+    // Set to true to start the highlights hidden. Use the command
+    // "sublime_linter_toggle_highlights" to toggle the highlights
```

Comment:
`sublime_linter_toggle_highlights` is indeed the name of the command, but we should assume users will call the command via the command palette, and therefor refer to it via the name we gave it there.

Reply:
So "SublimeLinter: Toggle Highlights"?

Reply:
I went ahead and changed it to that. Luckily the line is the same length after the edits 😉

Reply:
I guess without the SublimeLinter prefix would also work but 👍🏻